src/components/Dialog/DialogBody/DialogBody.tsx

```diff
@@ -9,10 +9,11 @@ const b = block('dialog-body');
 export interface DialogBodyProps {
     children: React.ReactNode;
     className?: string;
+    withDividers?: boolean;
 }
 
 export function DialogBody(props: DialogBodyProps) {
-    const {className} = props;
+    const {className, withDividers = false} = props;
```

Comment:
We use `has*` naming throughout the codebase (i.e. `hasClear`, `hasClose`, `hasArrow`)